Topic 3/src/main/java/bootcamp/exercises/ConvertToText.java

```diff
@@ -0,0 +1,130 @@
+package bootcamp.exercises;
+
+public class ConvertToText {
+	private static final String[] UNITS = { "", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine" };
+
+	private static final String[] TENS = { "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen",
+			"eighteen", "nineteen", "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety" };
+
+	static final int hundred = 100;
```

Comment:
[Medium] All constants should be in UPPERCASE. Also, add proper names. What is the meaning of each number here? Are they MAX_DIGITS, MIN_VALID_NUMBER, MAX_DOZEN_NUMBER?